app/views/mail_shared/_outlook_fix.html.erb

```diff
@@ -0,0 +1,1136 @@
+<style>
+img.no-float-image{
```

Comment:
via frontend style guide: Put spaces before { in rule declarations.

https://pages.18f.gov/frontend/css-coding-styleguide/format/

cc @msecret :yum: